### PR TITLE
SevenTV.Chatterino7 version 7.4.0

### PR DESCRIPTION
**Pull request by Vanidor**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95490)

### CODE REVIEW

**Comment by Vanidor** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @Vanidor,

The package installs normally, but requires `Microsoft.VCRedist.2015+.x64` (or `x86`) to run:
image

File: manifests/s/SevenTV/Chatterino7/7.4.0/SevenTV.Chatterino7.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SevenTV.Chatterino7
+PackageVersion: 7.4.0
```
Comment (stephengillie) [inline manifests/s/SevenTV/Chatterino7/7.4.0/SevenTV.Chatterino7.installer.yaml:5]:
```suggestion
PackageVersion: 7.4.0
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**